### PR TITLE
removed pinecone and anthropic from coming soon

### PR DESCRIPTION
**Pull request by vishnuszipstack**
## What

- Removed pinecone, supabase and anthropic from coming soon
- Updated readme

## Why

-

## How

-

## Database Migrations

- 

## Env Config

- 

## Relevant Docs

-

## Related Issues or PRs

-

## Dependencies Versions

-

## Notes on Testing

-

## Screenshots
image
image

## Checklist

I have read and understood the [Contribution Guidelines]().

### CODE REVIEW

File: frontend/src/components/input-output/list-of-sources/ListOfSources.jsx [hunk 1]
```diff
@@ -25,13 +25,9 @@ function ListOfSources({ setSelectedSourceId, type }) {
 
   const disabledIdsByType = {
     EMBEDDING: ["huggingface|90ec9ec2-1768-4d69-8fb1-c88b95de5e5a"],
-    LLM: [
-      "replicate|2715ce84-05af-4ab4-b8e9-67ac3211b81e",
-      "anthropic|90ebd4cd-2f19-4cef-a884-9eeb6ac0f203",
-    ],
+    LLM: ["replicate|2715ce84-05af-4ab4-b8e9-67ac3211b81e"],
     VECTOR_DB: [
       "supabase|e6998e3c-3595-48c0-a190-188dbd803858",
```
Comment (gaya3-zipstack) [inline frontend/src/components/input-output/list-of-sources/ListOfSources.jsx:30]: @vishnuszipstack we can remove Supabase from here. It has been disabled from adapters itself. So this filter will not kick in.
Also we need to change the README in this repo to reflect that we are supporting Pinecone and Anthropic.
CC: @nehabagdia

Comment (vishnuszipstack) [reply frontend/src/components/input-output/list-of-sources/ListOfSources.jsx]: done @gaya3-zipstack

File: frontend/src/components/input-output/list-of-sources/ListOfSources.jsx [hunk 2]
```diff
@@ -25,15 +25,8 @@ function ListOfSources({ setSelectedSourceId, type }) {
...
-      "replicate|2715ce84-05af-4ab4-b8e9-67ac3211b81e",
-      "anthropic|90ebd4cd-2f19-4cef-a884-9eeb6ac0f203",
-    ],
-    VECTOR_DB: [
-      "supabase|e6998e3c-3595-48c0-a190-188dbd803858",
-      "pinecone|83881133-485d-4ecc-b1f7-0009f96dc74a",
-      "milvus|3f42f6f9-4b8e-4546-95f3-22ecc9aca442",
-    ],
+    LLM: ["replicate|2715ce84-05af-4ab4-b8e9-67ac3211b81e"],
+    VECTOR_DB: ["milvus|3f42f6f9-4b8e-4546-95f3-22ecc9aca442"],
```
Comment (athul-rs) [inline frontend/src/components/input-output/list-of-sources/ListOfSources.jsx:29]: @vishnuszipstack Is supabase also removed ?

**Review by athul-rs — CHANGES_REQUESTED** [review]

File: frontend/src/components/input-output/list-of-sources/ListOfSources.jsx (hunk 2 above)
Comment (vishnuszipstack) [reply frontend/src/components/input-output/list-of-sources/ListOfSources.jsx:29]: done @athul-rs

**Review by athul-rs — APPROVED** [review]
LGTM

**Review by mohamed-siddhiq — APPROVED** [review]
LGTM

File: README.md [hunk 3]
```diff
@@ -71,18 +71,20 @@ Unstract comes well documented. You can get introduced to the [basics of Unstrac
...
 
 ### Vector Databases
 
 || Provider | Status |
 |---|---|---|
 |<img src="docs/assets/3rd_party/qdrant.png" width="32"/>| Qdrant | ✅ Working |
 |<img src="docs/assets/3rd_party/weaviate.png" width="32"/>| Weaviate | ✅ Working |
+|<img src="docs/assets/3rd_party/pinecone.png" width="32"/>| Pinecone | ✅ Working |
 |<img src="docs/assets/3rd_party/milvus.png" width="32"/>| Milvus | 🗓️ Coming soon! |
 |<img src="docs/assets/3rd_party/postgres.png" width="32"/>| PostgreSQL | 🗓️ Coming soon! |
```
Comment (gaya3-zipstack) [inline README.md:86]: @vishnuszipstack Having some confusion here. I think Postgres is  now officially supported. Does that mean we need to remove that img src also?

File: frontend/src/components/input-output/list-of-sources/ListOfSources.jsx [hunk 4]
```diff
@@ -25,15 +25,8 @@ function ListOfSources({ setSelectedSourceId, type }) {
...
-    LLM: [
-      "replicate|2715ce84-05af-4ab4-b8e9-67ac3211b81e",
-      "anthropic|90ebd4cd-2f19-4cef-a884-9eeb6ac0f203",
-    ],
-    VECTOR_DB: [
-      "supabase|e6998e3c-3595-48c0-a190-188dbd803858",
-      "pinecone|83881133-485d-4ecc-b1f7-0009f96dc74a",
-      "milvus|3f42f6f9-4b8e-4546-95f3-22ecc9aca442",
-    ],
+    LLM: ["replicate|2715ce84-05af-4ab4-b8e9-67ac3211b81e"],
```
Comment (gaya3-zipstack) [inline frontend/src/components/input-output/list-of-sources/ListOfSources.jsx:28]: Changes look good.

File: README.md (hunk 3 above)
Comment (vishnuszipstack) [reply README.md]: @gaya3-zipstack updated readme for postgres.

**Review by gaya3-zipstack — APPROVED** [review]
Looks fine.

**Pull request merged**